src/main/java/com/fasterxml/jackson/databind/DatabindException.java

```diff
@@ -39,10 +39,4 @@ protected DatabindException(String msg, Throwable rootCause) {
      * current path
      */
     public abstract void prependPath(Object referrer, String fieldName);
-
-    /**
-     * Method called to prepend a reference information in front of
-     * current path
-     */
-    public abstract void prependPath(Object referrer, int index);
```

Comment:
you can't remove public methods - they need to be deprecated and can be removed in future versions

Reply:
Thanks for the feedback. Shall I add a @Deprecated annotation on the `prependPath` method or leave it as is?

Reply:
@pjfanning: I added the @deprecated annotation on the abstract `prependPath` method. Please let me know if this is ok.

Reply:
you seem to have removed most of the changes related to prependPath - does this now need to be deprecated? Can't it just be left alone?

Reply:
@pjfanning: I left prependPath alone